app/post/directives/views/post-view-directive.js

```diff
@@ -20,17 +20,20 @@ function (
                 $scope.currentView = 'map';
             }
 
-            if (!PostViewHelper.isViewAvailable($scope.currentView)) {
-                $scope.unavailableView = $scope.currentView;
-                $scope.currentView = 'unavailable';
-            }
+            // Watch views, in case we get new feature config
+            PostViewHelper.isViewAvailable($scope.currentView).then(function (available) {
+                if (!available) {
```

Comment:
I don't have strong opinions either way. This is the only place we use isViewAvailable right now..